github_activity/github_activity.py

```diff
@@ -462,8 +462,27 @@ def generate_activity_md(
     data = get_activity(
         target, since=since, until=until, kind=kind, auth=auth, cache=False
     )
+
+    # Raise error if GitHub API returned no activity at all
```

Comment:
Noting this could be causing a change of behavior in the jupyter releaser, as noticed in some workflows today:

https://github.com/jtpio/jupyterlite-server-contents/actions/runs/20604073817/job/59175936047

```
ValueError: No activity found for jtpio/jupyterlite-server-contents between v0.1.1 and None.
```

Reply:
Since the releaser was likely not expecting this call to raise, wondering if this could be considered somewhat breaking?

Reply:
Ah - maybe, I didn't realize it was something that people were hitting. Maybe we need to think through the intended behavior better, can you open an issue for what you expect to happen vs what happens here?

Reply:
Thinking more about this: isn't the case of no activity a valid case?

Should this really raise if there is no data?

https://github.com/executablebooks/github-activity/blob/82fc6bd65fe622fb71659c952a4d781c97da283f/github_activity/github_activity.py#L466-L471